src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -625,6 +629,21 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     with self.argument_context('storage share') as c:
         c.argument('share_name', share_name_type, options_list=('--name', '-n'))
 
+    for item in ['create', 'delete', 'exists', 'list', 'show', 'update']:
+        with self.argument_context('storage share-rm {}'.format(item), resource_type=ResourceType.MGMT_STORAGE) as c:
+            c.argument('resource_group_name', required=False)
+            c.argument('account_name', storage_account_type)
+            c.argument('share_name', share_name_type, options_list=('--name', '-n'), id_part='child_name_2')
+            c.argument('share_quota', type=int)
```

Comment:
I will prefer use --quota to keep consistent with existing `az share create` command

Reply:
The help is from SDK: The maximum size of the share, in gigabytes. Must be greater than
                                   0, and less than or equal to 5TB (5120). For Large File Shares,
                                   the maximum size is 102400.

---
In reply to: [367792148](https://github.com/Azure/azure-cli/pull/11649#discussion_r367792148) [](ancestors = 367792148)